ppr-ui/src/components/unitNotes/UnitNoteReview.vue

```diff
@@ -147,10 +154,21 @@ export default defineComponent({
       setMhrUnitNoteRegistration({ key: 'submittingParty', value: val })
     }
 
+    const handleEffectiveDateUpdate = (val: string) => {
+      setMhrUnitNote({ key: 'effectiveDateTime', value: val })
+      // expiry date is 90 days from effective date
+      const expiryDateTime = new Date(new Date(val).getTime() + 90 * 24 * 60 * 60 * 1000).toISOString()
+      setMhrUnitNote({ key: 'expiryDateTime', value: expiryDateTime })
```

Comment:
The UI creates/sets the expiry date? I assumed it just passed the effectiveDateTime to the api 🤔

Reply:
CAUC can submit an expiry date, CAUE must submit an expiry date.

Reply:
Is there implications if the user modifies there local system time here?

Reply:
@cameron-eyds I thought so too, but then I looked at API spec again, and there was the expiry date that we need to submit, so I added it here. There may be implications if local time on machine changes.
Maybe we can pass a flag (and not the date/time string) that the effective date is immediate? Then based on that server generates effective and exp date? For past date, we can just pass a date/string in past. The only problem I see is that the user will not see the (server time) summary note about the Notice at the bottom of the component.